services/s3/config/crd/bases/s3.services.k8s.aws_buckets.yaml

```diff
@@ -1,10 +1,10 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.3.1-0.20200716001835-4a903ddb7005
+    controller-gen.kubebuilder.io/version: v0.4.0
```

Comment:
@jaypipes Do you think it is enough if annotated here?

Reply:
It's not really about the annotation (that is created by the `controller-gen crd` command). It's that we need to fix ALL the controllers, not just S3 :)

And the root of the fix is in how we install a specific version of `controller-gen`. We're currently using `go get` which has the very unfortunate side-effect of modifying the `go.mod` and `go.sum` file for the source repository, which isn't what we want.

I'll take a stab at trying to fix this in a less instrusive way, @ricardo-larosa.

Reply:
Cool! For reference https://github.com/kubernetes-sigs/kubebuilder/issues/1351